heron/stmgr/src/cpp/manager/stmgr.h

```diff
@@ -66,6 +67,11 @@ class StMgr {
                                proto::stmgr::TupleStreamMessage2* _message);
   void HandleInstanceData(sp_int32 _task_id, bool _local_spout,
                           proto::system::HeronTupleSet* _message);
+  // Called when an instance does checkpoint and sends its checkpoint
+  // to the stmgr to save it
+  void HandleStoreInstanceStateCheckpoint(sp_int32 _task_id,
```

Comment:
why are both instance and task id necessary? Doesn't the former contain the latter?